documentation/modules/proc-uninstalling-using-olm.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// assembly-uninstalling.adoc
+
+[id='uninstalling-olm-{context}']
+= Uninstalling {ProductName} using OLM
+
+This method uninstalls {ProductName} that was installed using the Operator Marketplace.
+
+.Procedure
+
+ifeval::["{cmdcli}" == "oc"]
+. Log in as a user with `cluster-admin` privileges:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} login -u system:admin
+----
+endif::[]
+
+. Remove all IoTProject and AddressSpace instances:
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+{cmdcli} delete iotprojects -A --all
+{cmdcli} delete addressspaces -A --all
```

Comment:
what is this -A? it doesn't work for me

Reply:
@famartinrh from oc delete --help:

```
  -A, --all-namespaces=false: If present, list the requested object(s) across all namespaces. Namespace in current
context is ignored even if specified with --namespace.
```

Which version of oc/kubectl are you using? Could be I need to change it to --all-namespaces=true, but I think its safe to require client >= 4.x

Reply:
my oc version is 3.11, I agree on requiring client >= 4 having that this is for OLM